src/Components/OATPropertyEditor/Internal/FormRootModelDetails/Internal/FormRootModelDetailsContent/PropertyDetailsEditorModalContent.tsx

```diff
@@ -656,7 +657,7 @@ const PropertyDetailsEditorModalContent: React.FC<IModalFormRootModelContentProp
                                                     draft
                                                 )
                                             ) {
-                                                // delete draft.minMultiplicity;
+                                                delete draft.minMultiplicity;
                                                 return draft;
```

Comment:
not in this PR, but do we need to "return" draft in produce?

Reply:
I think so, becaues the return of this function is what gets passed to set state. 🤔